packages/headless/src/features/commerce/did-you-mean/did-you-mean-slice.ts

```diff
@@ -24,6 +25,9 @@ export const didYouMeanReducer = createReducer(
           wordCorrections:
             queryCorrection?.corrections[0]?.wordCorrections ?? [],
         };
+      })
+      .addCase(applyCorrection, (state, action) => {
+        state.wasCorrectedTo = action.payload;
       });
```

Comment:
should I rename this action to `setWasCorrectedTo` instead of `applyCorrection` ?

Reply:
👎 :  consider the `state.wasCorrectedTo` as some internal part of Headless.
With that in mind, I think the `applyCorrection(correction)` express better what the action does for the implementers.